addons/sale_timesheet/wizard/sale_make_invoice_advance_views.xml

```diff
@@ -15,13 +15,23 @@
                     name="timesheet_invoice_date_range"
                     attrs="{'invisible': ['|', ('invoicing_timesheet_enabled', '=', False), ('advance_payment_method', '!=', 'delivered')]}"
                 >
+                    <label for="date_start_invoice_timesheet" string="Timesheets Period"/>
```

Comment:
Could just be a string in the field

Reply:
@xavierbol no that is not working.
![2023-12-05_16-19](https://github.com/odoo/odoo/assets/136676226/3d7bc3d4-2d63-41b0-85af-789992885fdf)

Reply:
it works, it is just because you added `nolabel="1"`... 😕 

I will fix it